views/js/qtiCreator/widgets/interactions/mediaInteraction/states/Question.js

```diff
@@ -297,6 +297,11 @@ define([
                     value = $.trim(attrValue).toLowerCase();
 
                     if (/^http(s)?:\/\/(www\.)?youtu/.test(value)) {
+                        if (attrValue.indexOf('&' > 0)) {
+                            youTubeUrl = new URL(attrValue);
+                            youTubeUrl.searchParams.delete('ab_channel');
```

Comment:
you need to delete all params except `v `

Reply:
https://caniuse.com/?search=searchParams is not supported in IE11

Reply:
I can see that , The npm package is tested in IE10+ (but Yes, you are correct i saw that in the Mozilla.org but i think the NPM pakcage had a fix for that ) , But i ill check it once more, 
![2021-05-25-14-02-www npmjs com](https://user-images.githubusercontent.com/74759660/119466292-19aaca00-bd62-11eb-8bed-347db7d716b1.png)

Reply:
I can see that , The npm package is tested in IE10+ (but Yes, you are correct i saw that in the Mozilla.org but i think the NPM package had a fix for that ) , But i ill check it once more, 
![2021-05-25-14-02-www npmjs com](https://user-images.githubusercontent.com/74759660/119466292-19aaca00-bd62-11eb-8bed-347db7d716b1.png)